cmd/backendapi/Dockerfile

```diff
@@ -19,5 +19,6 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo .
 
 FROM gcr.io/distroless/static
 COPY --from=builder /go/src/github.com/GoogleCloudPlatform/open-match/cmd/backendapi/backendapi .
+COPY --from=builder /go/src/github.com/GoogleCloudPlatform/open-match/api/protobuf-spec/backend.swagger.json .
```

Comment:
Add a TODO to remove this later. I'm going to go-bindata this.

Reply:
Gotcha, I am working on the go-bindata feature